proto/v1alpha1/rode_policy.proto

```diff
@@ -78,28 +78,42 @@ message ListPoliciesResponse {
 }
 
 message UpdatePolicyRequest {
-  // the auto-generate id of the occurrence
+  // the auto-generated id of the policy
   string id = 1;
-  
-  PolicyEntity policy = 2;
+  Policy policy = 2;
 
   // The fields to update.
-  google.protobuf.FieldMask update_mask = 3;
+  google.protobuf.FieldMask update_mask = 4;
+}
+
+message Policy {
+  // Unique autogenerated id
+  string id = 1;
+  // The name of the policy
+  string name = 2;
+  // A brief description of the policy
+  string description = 3;
+  // The current policy version -- not necessarily the newest.
+  int32 current_version = 4;
+  // Current policy code or source. The policy.version matches current_version
+  // This is set by Rode, and isn't meant to be populated by users
+  PolicyEntity policy = 5;
```

Comment:
I added a `policy` field here which will have the `currentVersion` of the policy as its content. The `policy` field is unset before it's saved into Elasticsearch, so the field is technically there but the content isn't. 

The alternatives would be to duplicate the `rego_content`/`source` fields on the unversioned parent policy or have require callers to make a second call to fetch the policy content by its version.

Reply:
this approach makes sense to me